Content/Items/UndergroundTemple/Accessories.TempleLensUpgrade.cs

```diff
@@ -31,13 +33,20 @@ public override void Load()
 		private void ModifyHurtLens(Player Player, Item Item, NPC target, ref int damage, ref float knockback, ref bool crit)
 		{
 			if (Equipped(Player) && crit)
-				target.AddBuff(BuffType<Exposed>(), 120);
+				ApplyExposed(target);
```

Comment:
Should we consider switching over to the exposure system added by Skippz a while back? That way we can take advantage of proper stacking and keywords.

Reply:
Seems like more of a design discussion. Ping me on the dev discord when you read this and we can discuss with egshels